### PR TITLE
link add task form to update database

### PR DESCRIPTION
**Pull request by gene83**

### CODE REVIEW

File: migrations/3_20190212095852_cards.js [hunk 1]
```diff
@@ -16,8 +16,8 @@ exports.up = function(knex, Promise) {
     table
       .integer('created_by')
       .references('id')
-      .inTable('users')
-      .notNullable();
+      .inTable('users');
+    // .notNullable();
```
Comment (taesup) [inline migrations/3_20190212095852_cards.js]: Any reason why this is nullable?

File: migrations/3_20190212095852_cards.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ exports.up = function(knex, Promise) {
   return knex.schema.createTable('cards', table => {
     table.increments();
     table.string('title').notNullable();
-    table.string('body', 1024).notNullable();
+    table.string('body', 1024);
```
Comment (taesup) [inline migrations/3_20190212095852_cards.js:5]: Have you already run this migration? Migration files should not be edited once they are created and run. The proper solution is to create a new migration file that fixes anything that was wrong with the old one, using tools like alterColumn or alterTable.

File: server/server.js [hunk 3]
```diff
@@ -80,6 +81,27 @@ passport.use(
   })
 );
 
+app.post('/cards', (req, res) => {
+  console.log(req.body);
```
Comment (taesup) [inline server/server.js]: Remove console.log

File: server/server.js [hunk 4]
```diff
@@ -80,6 +81,27 @@ passport.use(
...
+
+  const newCard = {
+    title: req.body.title,
+    priority_id: parseInt(req.body.priority),
+    status_id: 1
+  };
+
+  console.log(newCard);
```
Comment (taesup) [inline server/server.js]: Remove console.log

**Review by taesup — CHANGES_REQUESTED** [review]

File: migrations/3_20190212095852_cards.js (hunk 2 above)
Comment (gene83) [reply migrations/3_20190212095852_cards.js:5]: created new migration to return this to notnullable in commit below

File: migrations/3_20190212095852_cards.js (hunk 1 above)
Comment (gene83) [reply migrations/3_20190212095852_cards.js]: leaving as nullable for now, for easier route testing before integrating users

Comment (taesup) [inline migrations/3_20190212095852_cards.js]: You should get rid of the comment here still.

Comment (taesup) [reply migrations/3_20190212095852_cards.js]: It's still showing up as if you're changing this migration.

**Review by taesup — CHANGES_REQUESTED** [review]

**Review by taesup — APPROVED** [review]
LGTM!

**Pull request merged**